cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -148,7 +148,7 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
                 do discardsInBranch.%ToJSON(%data)
                 set handled = 1
             } elseif (pathStart = "contexts") {
-                set contexts = ##class(SourceControl.Git.Utils).GetContexts()
+                set contexts = ##class(SourceControl.Git.Utils).GetContexts(1)
```

Comment:
I think this change also impacts the WebUI - could make this subject to a URL parameter so the WebUI continues to function as-is but we don't try to git pull in non-namespace contexts.